src/components/InitialComponents/LegislatorHomePage/index.js

```diff
@@ -0,0 +1,197 @@
+          profile: requirement.profile.photo,
+          name: requirement.profile.name,
+          message: comment,
+        };
+    
+        if (comment) {
+          setRequirement({
+            ...requirement,
+            comments: [...requirement.comments, data],
+          });
+          setComment("");
+        }
+      };
+
+    return (
+        <div>
+            <Menu />
+
+            <HeaderBar />
+                      
+            <LegislatorCard requeriments={299} analysis={274} denied={10}
+            done = {15}>  </LegislatorCard>
+
+            <section className="requirement_container"> 
+                <section  className="requirement_content"  onClick={() => modalRef.current.openModal()}>
+                    
+                    <div className="header_title">
+                        <p>Reivindicação em destaque</p>
+                        <Box className="status">
+                            <span>{requirement.status}</span>
+                        </Box>
+                    </div>
+                        
+                    <div className="header_profile">
+                                                             
+                            <Box className="profile">
+
+                                { requirement.profile.photo ? (
+                                <img
+                                    src={requirement.profile.photo}
+                                    width={30}
+                                    height={30}
+                                    alt="Image"
+                                />
+                                    ) : (
+                                    <AccountCircleRoundedIcon color="action" />
+                                )}
+
+                                <Box className="info">
+                                    <p>Nome: <strong>{requirement.profile.name}</strong></p>
+                                    <p>Cidade: <strong>{requirement.profile.city}</strong></p> 
+                                </Box>
+
+                            </Box>
+
+                            <h5 className="date">
+                                    <strong> Data de publicação:  </strong>
+                                    <p>{requirement.profile.dateOccurrence}</p>
+                            </h5>                      
+                    </div>
+
+                    <p className="description"> {requirement.message}</p>
+
+        
+                </section>
+
+                <Box className="box_support">
+                        <Button onClick={() => handleSupport()}>
+                            {support ? (
+                            <ThumbUpAltRoundedIcon />
+                            ) : (
+                            <ThumbUpAltOutlinedIcon color="action" />
+                            )}
+                        </Button>
+                       
+                        <Button onClick = {() => handleShowComments()}> <CommentIcon /></Button>
```

Comment:
```suggestion
    <Button onClick={() => handleShowComments()}>
        {showComments ? <CommentIcon /> : <CommentOutlinedIcon />}
    </Button>
```

É interessante distinguir a cor do botão para quando o modal estiver aberto ou fechado.

Antes: 
<img src="https://user-images.githubusercontent.com/56003521/159007560-54a35d63-d213-409c-82d3-6073f929e869.png" />

Depois: 
<img src="https://user-images.githubusercontent.com/56003521/159007597-423e50fb-5745-43b8-a853-51122dee19ff.png" />

Reply:
É interessante que o input esteja dentro do card, e tenha um padding para não ficar tão colado nas bordas.

<img src="https://user-images.githubusercontent.com/56003521/159008763-ce4aa16c-8d1c-436a-8838-d9a6648af230.png" />